docs/fsharp/index.yml

```diff
@@ -0,0 +1,68 @@
+### YamlMime:Landing
+
+title: "F# documentation"
+summary: "Learn how to write any application using the F# programming language on .NET."
+
+metadata:
+  title: "F# docs - get started, tutorials, reference."
+  description: "Learn F# programming - for developers new to F#, and experienced F# / .NET developers"
+  ms.topic: landing-page # Required
+  author: cartermp
+  ms.author: phcart
+  ms.date: 11/25/2019
+
+landingContent:
+  - title: "Learn to program in F#"
+    linkLists:
+      - linkListType: get-started
+        links:
+          - text: "What is F#?"
+            url: what-is-fsharp.md
+          - text: "Install F#"
+            url: get-started/install-fsharp.md
+          - text: "Introductory tutorials"
+            url: tutorials/intro-to-csharp/index.md
+          - text: "Create C# apps with Visual Studio"
```

Comment:
This seems wrong. Is there an equivalent F# page?

Reply:
YOikes, will fix